ingestion/setup.py

```diff
@@ -176,6 +176,7 @@ def get_long_description():
         "Source": "https://github.com/open-metadata/OpenMetadata",
     },
     packages=find_namespace_packages(where="./src", exclude=["tests*"]),
+    namespace_package=["metadata"],
```

Comment:
Could you please expand on why are we adding this?

Reply:
Because of This https://github.com/open-metadata/OpenMetadata/issues/6560 Issue.